### PR TITLE
Fix slow tests v4.2.0

### PR DESCRIPTION
**Pull request by LysandreJik**
Fixes a bunch of slow tests that were failing.

### CODE REVIEW

File: tests/test_modeling_encoder_decoder.py [hunk 1]
```diff
@@ -822,7 +822,9 @@ def prepare_config_and_inputs(self):
         }
 
     def get_pretrained_model(self):
-        return EncoderDecoderModel.from_encoder_decoder_pretrained("bert-large-uncased", "prophetnet-large-uncased")
+        return EncoderDecoderModel.from_encoder_decoder_pretrained(
```
Comment (patrickvonplaten) [inline tests/test_modeling_encoder_decoder.py:825]: Thanks a lot!

**Review by patrickvonplaten — APPROVED** [review]
Thanks a lot for fixing the tests!

File: .github/workflows/self-scheduled.yml [hunk 2]
```diff
@@ -75,7 +75,7 @@ jobs:
           RUN_SLOW: yes
         run: |
           source .env/bin/activate
-          pip install -r examples/requirements.txt
+          pip install -r examples/_tests_requirements.txt
```
Comment (sgugger) [inline .github/workflows/self-scheduled.yml:78]: Good catch!

**Review by sgugger — APPROVED** [review]
Thanks for all the fixes, you rock!

**Pull request merged**